CMSgov/mint-app
# [EASI-3016] - Get access option homepage

# EASI-3016

[Figma](https://www.figma.com/file/aD3DZLft8HZXqJjWCIT3i2/MINT-Branding-%26-Marketing?type=design&node-id=809-7026&t=Iabx51pcPlY5IJnb-0)

## Changes and Description

- Added links on landing page to navigate to access article
- View for how to get access article
- Info alert on sign in with link to article
- Custom hook for USWDS in-page navigation react integration
  - Hook explanation in code comments
  - 
![Screenshot 2023-06-22 at 11 39 50 AM](https://github.com/CMSgov/mint-app/assets/95709965/a868c0a5-1da3-4ca5-80d3-0997139cc3c5)

## How to test this change

- Navigate to landing page
- Click the links that navigate to the access article
- Verify copy changes

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
